### ISSUE TITLE
5.4.0 breaks using SequelizeJSON in the where parameter

### ISSUE THREAD

**Issue by mlocatelli-pge** [mickhansen/graphql-sequelize#508]
In 5.3.3, a query similar to the following works without any issue
```
query ($offset: Int,
       $limit: Int,
       $order: String,
       $whereClause: SequelizeJSON) {
  asset: some_table (offset: $offset,
                 limit: $limit,
                 order: $order,
                 where : $whereClause) {
     someValue1
     someValue2
    }
}
```
When using 5.4.0, I'm getting the following errors:
1) When the $whereClause variable is an empty object {}
`"message": "Variable \"$whereClause\" got invalid value {}.\nExpected type \"SequelizeJSON\", found {}: Unexpected token o in JSON at position 1"`
2) When the $whereClause variable has something in it
`"message": "Variable \"$whereClause\" got invalid value {\"someValue1\":{\"gte\":10,\"lte\":50}}.\nExpected type \"SequelizeJSON\", found {\"someValue1\":{\"gte\":10,\"lte\":50}}: Unexpected token o in JSON at position 1"`

**Comment by Alexsey** [comment]
It will work if you provide a JSON string value of a SequelizeJSON parameter type. Could you provide a full code snippet that was working in 5.3.3 an been broken in 5.4.0?

**Comment by RichAyotte** [comment]
In `5.4.0` I have to `JSON.stringify(where)` which wasn't required in 5.3.3.

Is this a new requirement or a regression?

**Comment by mickhansen** [comment]
Seems like a regression @RichAyotte

**Comment by mlocatelli-pge** [comment]
Awesome! #511 seems to fix this with version `5.4.2`
